capability_linux.go

```diff
@@ -285,15 +286,12 @@ func (c *capsV3) Load() (err error) {
 		return
 	}
 
-	var status_path string
-
-	if c.hdr.pid == 0 {
-		status_path = fmt.Sprintf("/proc/self/status")
-	} else {
-		status_path = fmt.Sprintf("/proc/%d/status", c.hdr.pid)
+	path := "/proc/self/status"
+	if c.hdr.pid != 0 {
+		path = "/proc/" + strconv.Itoa(int(c.hdr.pid)) + "/status"
```

Comment:
nit: I think Sprintf looks better for this case.

Reply:
Yep; fixed